migrations/1552064796155-tx-paper-wallet.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const db = require('./db')
+
+exports.up = function (next) {
+  var sql = [
+    'ALTER TABLE cash_in_txs ADD COLUMN is_paper_wallet boolean null DEFAULT false',
```

Comment:
Is this being used?

Reply:
Yes, the [migration script](https://github.com/lamassu/lamassu-server/blob/dev/bin/lamassu-migrate) goes through all the scrips in the migration directory.

Reply:
I mean is the ``is_paper_wallet`` field ever being set?

Reply:
Yes, it is. [See here](https://github.com/lamassu/lamassu-machine/blob/449a6cae14d97aa875281cd72372bd040a554699/lib/brain.js#L1775).